apps/datafeeder/src/app/presentation/components/upload-data-rules/upload-data-rules.component.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+
+import { UploadDataRulesComponent } from './upload-data-rules.component'
+import { By } from '@angular/platform-browser'
+import { TranslateModule } from '@ngx-translate/core'
+
+describe('UploadDataRulesComponent', () => {
+  let component: UploadDataRulesComponent
+  let fixture: ComponentFixture<UploadDataRulesComponent>
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [TranslateModule.forRoot()],
+      declarations: [UploadDataRulesComponent],
+    }).compileComponents()
+  }))
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(UploadDataRulesComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
+    expect(component).toBeTruthy()
+  })
+
+  it('should display accepted file formats', () => {
+    const el = fixture.debugElement.query(By.css('.formats')).nativeElement
+
+    expect(el.childElementCount).toEqual(component.acceptedFileFormats.length)
+
+    const childArr = [...el.children]
+    childArr.forEach((c) => {
+      expect(
```

Comment:
What does it test ?

Reply:
This test ensures that in a <span> elements will be display accepted file formats from input